src/language-css/printer-postcss.js

```diff
@@ -679,6 +679,7 @@ function genericPrint(path, options, print) {
               )
             ])
           ),
+          node.groups.length > 1 && options.trailingComma !== "none" ? "," : "",
```

Comment:
Can we add check what it is `isSCSS()`, in theory we can break less code, some projects can have scss/less files togethe (legacy or migration period)

Reply:
@ad1992 nested maps, when variable contain map and each key contain other map/list

Reply:
sorry i guess there is some confusion. i understood about adding nested map and lists. But i didnt get `Can we add check what it is isSCSS(), in theory we can break less code, some projects can have scss/less files togethe (legacy or migration period`

So is there any specifice check you meant in the above comment  except for nested map and lists ?

Reply:
@ad1992 parsers some times change ast and introduce new nodes, better check directly what is `SCSS` to avoid break something in future

Reply:
@evilebottnawi so you mean we need to make sure trailing comma is added only in case of  `scss`. 
I referred to `parser-postcss.js` file and added check for `isSCSS`. 
Is this fine ? 
Sorry since I am new to prettier codebase and AST, I am asking too many questions.